### PR TITLE
Update README for new lv_img_conv

### PR DESCRIPTION
**Pull request by FintasticMan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,10 +57,13 @@ Then install the `lv_font_conv` executable to the InfiniSim source directory (wi
...
 ```
 
-When you want to create a `resource.zip` file then install the `lv_img_conv` executable to the InfiniSim source directory (will be installed at `node_modules/.bin/lv_img_conv`)
+When you want to create a `resource.zip` file then install the `pillow` Python library to the InfiniSim source directory (will be installed in `.venv/`)
 
 ```sh
-npm install ts-node@10.9.1 @swc/core lv_img_conv@0.3.0
+python3 -m venv .venv
+source .venv/bin/activate
+pip install wheel
```
Comment (NeroBurner) [inline README.md:65]: do we need to first install the wheel package? can't wheel and Pillow be installed at the same time?

**Review by NeroBurner — COMMENTED** [review]
thanks a lot for updating the readme! just one question before I merge

File: README.md (hunk 1 above)
Comment (FintasticMan) [reply README.md:65]: I'm honestly not sure, I'll try it out in a bit.

Comment (FintasticMan) [reply README.md:65]: It seems like you can do them in 1 command

**Review by NeroBurner — APPROVED** [review]
Thanks!

**Pull request merged**